mono/metadata/mono-perfcounters.h

```diff
@@ -8,18 +8,32 @@
 #include <glib.h>
 #include <mono/metadata/object.h>
 #include <mono/utils/mono-compiler.h>
+#include <mono/metadata/handle.h>
 
 typedef struct _MonoCounterSample MonoCounterSample;
 
+/* old names:
```

Comment:
You can just delete the old declarations. They'll still be in the git history if we ever need them

Reply:
Git history is much less discoverable than comments in live code.